### PR TITLE
Doc: Pacemaker Explained: convert "Advanced Configuration" chapter to rST+sphinx

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review?

File: doc/sphinx/Pacemaker_Explained/advanced-options.rst [hunk 1]
```diff
@@ -1,776 +1,766 @@
...
+          <operations>
+            <op id="resHealthIOWait-monitor-interval-5" interval="5" name="monitor" timeout="5"/>
+            <op id="resHealthIOWait-start-interval-0s" interval="0s" name="start" timeout="10s"/>
+            <op id="resHealthIOWait-stop-interval-0s" interval="0s" name="stop" timeout="10s"/>
+          </operations>
+        </primitive>
+      </clone>
+
+This way ``attrd_updater`` will set proper status for each node running this
+resource. Any attribute matching "#health-[a-zA-z]+" will force the cluster to
```
Comment (clumens) [inline doc/sphinx/Pacemaker_Explained/advanced-options.rst:694]: Putting the regex in monospace might be nice to set it off from the surrounding text.

Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/advanced-options.rst:694]: Good point, and that made me realize it didn't quite look right ... sure enough, all we check is that it starts with "#health" (we don't even check for the dash currently).

Comment (kgaillot) [reply doc/sphinx/Pacemaker_Explained/advanced-options.rst]: Updated and reworded

**Comment by clumens** [comment]
Looks good to me.

**Pull request merged**